### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Anshuraja01**
I made changes

### CODE REVIEW

**Comment by firdoush01** [comment]
thank u bkl for making changes

**Comment by Anshuraja01** [comment]
Okkk

File: index.html [hunk 1]
```diff
@@ -6,8 +6,13 @@
     <title>Document</title>
 </head>
 <body>
-    Hello world
+    Hello Dc
+
+    How are you  
+
+
+    Sb thikk?????
```
Comment (firdoush01) [inline index.html:14]: yhi sb pucha jata h

File: index.html [hunk 2]
```diff
@@ -6,8 +6,13 @@
     <title>Document</title>
 </head>
 <body>
-    Hello world
```
Comment (firdoush01) [inline index.html:9]: delete q kia h pucha h hmse

Comment (Anshuraja01) [reply index.html:9]: **puchne ka kya jarurat h**

**Review by firdoush01 — COMMENTED** [review]
This is the final addition ?

**Pull request merged**